Sources/Spezi/Spezi.docc/Spezi_Ecosystem.md

```diff
@@ -0,0 +1,180 @@
+# The Spezi Ecosystem
+
+<!--
+This source file is part of the Stanford Spezi open-source project
+SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+SPDX-License-Identifier: MIT
+-->
+
+Discover the core modules available in the Spezi framework and understand their use cases.
+
+## Overview
+
+The Spezi ecosystem provides a collection of interoperable modules designed to accelerate healthcare and research application development. Each module is built to work independently or in combination with others, allowing you to choose the components that best fit your needs.
+
+## Core Modules
+Some of the following modules can be tested and interact with using the Template Application: **[SpeziTemplate](https://github.com/StanfordSpezi/SpeziTemplateApplication)**
+
+### Spezi Onboarding
+- **[SpeziOnboarding](https://github.com/StanfordSpezi/SpeziOnboarding)**
+    - [SpeziOnboarding Documentation](https://swiftpackageindex.com/StanfordSpezi/SpeziOnboarding/1.2.2/documentation/spezionboarding) 
```

Comment:
We might want to package these documentation links into nice Tip boxes or something similar that highlights them in the DocC documentation? In addition, I would avoid linking concrete versioned tags, we should always aim to link to the latest documentation (https://swiftpackageindex.com/StanfordSpezi/SpeziOnboarding/1.2.2/documentation/spezionboarding -> https://swiftpackageindex.com/StanfordSpezi/SpeziOnboarding/documentation/spezionboarding). Links without explicit version tags should always point to the latest version.